### DISCUSSION TITLE
Has anyone had success with solution-config.props file

### DISCUSSION THREAD

**Discussion by randomroxks** [unoplatform/uno#15948] [Q&A]
So, I'm using the solution-config.props.config tip to improve the build times. For reference its here: 
Improve performance using the Uno Platform templates
When using an Uno Platform 5.0 template, in the Solution Explorer, you'll find a folder named Solution Items, and a file named solution-config.props.sample.

You can follow the directions specified in this file, or follow them here:

Right-click on the root item (your solution name) and Open in file explorer
Locate the solution-config.props.sample and make a copy named solution-config.props in the same folder
Back in Visual Studio, and for your convenience you can right-click on Solution Items and add solution-config.props
Now that we have configured this file, let's say that you want to debug for WebAssembly:

In the solution-config.props file, uncomment the line that mentions Wasm or WebAssembly then save it.
For the change to take effect, close and reopen the solution, or restart Visual Studio.
Repeat this process when changing your active development platform.

When I un-comment iOS and add condition like this: 
`<OverrideTargetFramework Condition="$(RuntimeIdentifier.StartsWith('ios'))">$(DotNetVersion)-ios</OverrideTargetFramework>`

I get an error at build time:
OverrideTargetFramework set to 'net8.0-ios' is missing valid target. Set OverrideTargetFramework to one of the TargetFrameworks for this project or skip building this project (eg unload the project in Visual Studio)

but it is defined as one of the targets in the csproj file. 
`<TargetFrameworks>$(DotNetVersion)-android;$(DotNetVersion)-ios;$(DotNetVersion)-maccatalyst</TargetFrameworks>
    <TargetFramework Condition="'$(OverrideTargetFramework)'!=''">$(OverrideTargetFramework)</TargetFramework>
    <SingleProject>true</SingleProject>
    <OutputType>Exe</OutputType>`

Does anyone has any idea why am I getting the invalid target message?

**Comment by jeromelaban** [comment]
@Darsh0307 can you try this please?

**Reply by xperiandri** [reply]
I have the same question. How to enable a single target framework for development purposes with Uno 5.2 single project

**Reply by xperiandri** [reply]
Does the location of the file matter? Now it is in my solution root, while Uno projects are in a solution folder 1 level from it

**Reply by jeromelaban** [reply]
The support was removed from the project files to make the project simpler, but you can use the steps that are being updated in this PR: https://github.com/unoplatform/uno/pull/17229.

**Reply by xperiandri** [reply]
Thanks! Now I know that the trick is
`<TargetFrameworks Condition=" '$(OverrideTargetFramework)' != '' ">$(OverrideTargetFramework)</TargetFrameworks>`
